extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -199,6 +202,8 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.analyticsGroup_ =
         this.instrumentation_.createAnalyticsGroup(this.element);
 
+    this.handleRequestsFrameUrl_();
```

Comment:
This can be triggered from buildCallback if the amp-analytics element has attribute trigger=immediate but we definitely cannot modify the page at that point.  Instead we should do this explicitly as part of layoutCallback.  Also we need to find a way to "deprioritize" the frame creation as current code would give it priority of 1 while we want only the frame creation to be 2.  This will need some discussion with Dima.